src/main/java/com/stud/java/entityGenerators/BaseEntityGenerator.java

```diff
@@ -0,0 +1,37 @@
+package com.stud.java.entityGenerators;
```

Comment:
Naming of packages should not contain camelCase
It is better to divide such names with dots, like entity.generators

But in this specific case, you should place generators in com.stud.java.service.generator